src/scss/constants.scss

```diff
@@ -271,6 +271,11 @@ $z-layers: (
   }
 }
 
+@mixin circle-avatar {
+  object-fit: cover;
+  object-position: top;
+}
+
```

Comment:
please, also use this `@mixin` in the `ProfileModalAvatar` file.

Reply:
fixed in 026c1577db0e31821fc49de43e520395b32f9063